core/types.go

```diff
@@ -192,6 +192,12 @@ func (c Certificate) Chain(no int) []byte {
 	return bytes.Join(chain, nil)
 }
 
+type RevokedCertificate struct {
```

Comment:
Could you add a short godoc comment of the form "// RevokedCertificate is ...." to this type?

Reply:
Done.